MaxText/configs/models/deepseek3-671b.yml

```diff
@@ -34,5 +34,13 @@ shared_experts: 1
 routed_scaling_factor: 2.5
 routed_score_func: "sigmoid"
 routed_bias: True
-rope_max_timescale: 10_000
+max_target_length: 16384 # 4096 * 4
```

Comment:
Do we need this in a model config? Or we could just pass in the script during the run?

Reply:
DeepSeek provides the default in model config https://huggingface.co/deepseek-ai/DeepSeek-V3-Base/blob/main/inference/model.py#L22 but yes, we can just provide as an argument during training or inference. I will remove it.

Reply:
I think those are default values. For instance, it has `vocab_size: int = 102400`, while the DeepSeek's vocab size is 129280. Yeah, let's remove this config from model config.